ilc/client/pageTransitions.js

```diff
@@ -0,0 +1,59 @@
+let globalSpinner;
+const runGlobalSpinner = () => {
```

Comment:
This function can be called multiple times during state transition... But I don't see and code that would allow it to handle this correctly (w/o multiple timeouts/DOM nodes creation).